test/e2e/geojson/mapml2geojson.html

```diff
@@ -0,0 +1,288 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>MapML 2 geoJSON</title>
+  <meta charset="UTF-8">
+  <script type="module" src="mapml-viewer.js"></script>
+  <script src="lib/geojson.js"></script>
+</head>
+
+<body>
+  <!--All the mapml layers- that will be converted to geojson go in the mapml-viewer-->
+    <mapml-viewer projection="OSMTILE" zoom="10" lon="-75.7" lat="45.4" controls>

```

Comment:
I'm guessing that the mapml-viewer isn't necessary and that the tests might run a bit faster without it.